src/api/ruleData/ruleData.service.ts

```diff
@@ -30,10 +30,14 @@ export class RuleDataService {
 
   async createRuleData(ruleData: RuleData): Promise<RuleData> {
     try {
+      if (!ruleData._id) {
+        ruleData._id = new Date().getTime().toString();
```

Comment:
Would the _id of ruleData benefit from using mongoDB's ObjectID for making new IDs?
```
 import { ObjectId } from 'mongodb';

const newRuleID = new ObjectId();
const ruleData._id = newRuleID.toHexString();
```

Reply:
Yeah, good point. I did think about that, but the existing rules would have to be updated because their ids came from GoRules and are strings. So we'll have to manually go through and update them on The Hive afterward. Now that I'm thinking about it again I think that's worth it though. So I'll update this.

Reply:
Just realized you were actually suggesting a middleground where we get the string from the ObjectId. I like that! Updated!